lib/domains/uj/edu/sa/.txt

```diff
@@ -0,0 +1 @@
+University of Jeddah
```

Comment:
Add files via upload
https://www.uj.edu.sa/en 
It's Saudi Uni.
https://cs.uj.edu.sa/en/college-computer-science-and-engineering?programId=64
 [M-AlGhazali-patch-1](https://github.com/M-AlGhazali/Fork_Muha/compare/M-AlGhazali-patch-1) ([#1](https://github.com/M-AlGhazali/Fork_Muha/pull/1)) [M-AlGhazali-patch-2](https://github.com/M-AlGhazali/Fork_Muha/compare/M-AlGhazali-patch-2)